extensions/wikia/DesignSystem/i18n/zh-hant.json

```diff
@@ -131,5 +131,5 @@
   "wds-avatar-badges-global-discussions-moderator-tooltip": "全域話題版主",
   "wds-avatar-badges-helper-tooltip": "FANDOM助手",
   "wds-avatar-badges-staff-tooltip": "FANDOM工作人員",
-  "wds-avatar-badges-vstf-tooltip": "反破壞小組"
-}
\ No newline at end of file
+  "wds-avatar-badges-soap-tooltip": "反破壞小組"
```

Comment:
Are we sure about this translations? 😄

Reply:
nope, I've only changed key and will create ticket for translate to change wds-avatar-badges-soap-tooltip value from correct one (english)